moabb/pipelines/multi_pass.py

```diff
@@ -0,0 +1,111 @@
+from sklearn.base import BaseEstimator, TransformerMixin, clone
+from sklearn.feature_selection import SelectKBest, mutual_info_classif
+from pyriemann.spatialfilters import CSP
+from pyriemann.estimation import Covariances as Cov
+import numpy as np
+import logging
+
+log = logging.getLogger()
+
+
+class LogVariance(BaseEstimator, TransformerMixin):
+
+    def fit(self, X, y=None):
+        """fit."""
+        return self
+
+    def transform(self, X):
+        """transform, concatenating all frequency bands."""
+        assert X.ndim == 4
+        return np.concatenate([np.log(np.var(X[..., i], -1)) for i in range(X.shape[-1])], axis=1)
+
+
+class MultibandCovariances(BaseEstimator, TransformerMixin):
```

Comment:
I would suggest to create a `FilterBank` estimator, that takes any transformer in the constructor, and iterate over the last dimension and stack the output. This way we are not forced to create one specific estimator for every multiband estimator

Reply:
Good idea, will do

Reply:
Will fix this and the filtering bug in my next commit